templates/quickstart-vpc-for-atlassian-services.yaml

```diff
@@ -150,6 +157,11 @@ Conditions:
   GovCloudCondition: !Equals
     - !Ref 'AWS::Region'
     - us-gov-west-1
+  UsingDefaultBucket:
+    !Equals [!Ref QSS3BucketName, 'aws-quickstart']
+  NoBucketRegionSuppliedOrBucketPrefixIsDefault: !Or
+    - !Equals [!Ref QSS3BucketRegion, '']
+    - !Equals [!Ref QSS3KeyPrefix, 'quickstart-atlassian-services/']
```

Comment:
I don't quite understand this condition, why does it matter if the prefix is different whether we use a custom region or not? If I put in a custom bucket region but use the default key prefix it won't respect my custom bucket region will it?

Reply:
I figured this out from the other template. Because we now pass the key prefix down to the nested template, if we use the default key prefix we will get the template from the Atlassian services quickstart. Otherwise, we get the template from the product quickstart's nested submodule.